### DISCUSSION TITLE
Eufy Smart Floodlight

### DISCUSSION THREAD

**Discussion by barbatrukko** [indomus/forum#327] [HUB: Home Assistant]
Salve,
vorrei acquistare questo prodotto [Eufy Smart Floodlight ](https://us.eufylife.com/collections/smart-floodlight), ma ho letto in giro che è molto sensibile come sensore e si accende spesso.
Ho visto che a tutt'oggi (correggetemi se sbaglio) non esistono integrazioni per i prodotti eufy della linea security.
Chissà però se almeno la gestione dei faretti e del sensore di movimento sia visibile in HA?
Oppure prodotto equivalente da voi testato che si integra bene su HA?

**Comment by indomus** [comment]
Queste non le abbiamo testate. Su EZVIZ al massimo puoi valutare [queste](https://amzn.to/3otvkZ2), che si integrano molto bene con HA.

**Reply by barbatrukko** [reply]
Interessante. Chissà se è supportata dal mio NAS QNAP, la eufy ci registrava...
Quindi la posso usare per quello che è in più posso controllarla anche da HA. Quindi potrei ad esempio con dei parametri booleani di HA andare in override dei comandi del suo sensore e quindi accendere comunque i fari o spegnerli comunque?

**Reply by indomus** [reply]
leggi [qui](https://indomus.it/news/hot-aggiornamento-2021-5-per-home-assistant-core/): come vedi, la nuova integrazione EZVIZ prevede tutta una serie di sensori e switch, credibilmente anche quelli della LC1C

**Reply by barbatrukko** [reply]
Moooolto interessante. Studio un pochino e nel caso procedo con l'acquisto di EZVIZ allora. Tanto ha le stesse funzionalità di eufy ma è di sicuro meglio supportato.
Grazie!

**Reply by indomus** [reply]
Non si capiva perché ci mettessero tanto a riattivare l'integrazione EZVIZ... s'è capito quando l'hanno rientrodotta, è magnifica :D
